scripts/publish-functions.sh

```diff
@@ -43,9 +43,6 @@ for f in *; do
         (
         cd "$f" || exit 1
         echo "$f"
-        # Javascript does not allow function names with '-' so we need to
-        # replace the directory name (which might have '-') with "_"
-        functionname=${f//-/_}
```

Comment:
Without `functionname` set, this will cause the `gcloud functions deploy "$functionname"...` to fail a few lines down, perhaps replace with `functionname=${f}`?